Livecheckables/vim.rb

```diff
@@ -0,0 +1,4 @@
+class Vim
+  livecheck :url   => "https://github.com/vim/vim.git",
+            :regex => /^v?(\d+(?:\.\d+)+\d{2}(?<=00|50))$/
```

Comment:
```suggestion
            :regex => /^v?(\d+(?:\.\d+)+(?:00|50))$/
```

In my testing, `(?:00|50)` matched the same versions without being as verbose or requiring a lookbehind, so this could have been simplified if the PR was appropriate for livecheck.

Reply:
Thanks for the feedback @samford -- I was not sure if the `$` (which as you said would ensure that it did not match a tag like `1.2.3-rc1`) would work all the time with just a `(?:00|50)`, hence why I used a lookbehind with `\d{2}`to ensure that it performed the lookbehind after digits only. More to learn :)

Reply:
`$` basically means "the end of the line", which is also the end of the string in the context of Git tags. `(?:00|50))$` ensures that the last part of the string is `00` or `50` and since there's nothing between `\d+(?:\.\d+)+` and `(?:00|50)`, it would always be the last digits of the last part of the version (e.g., 1.2.300, 1.2.350, 1.2.3.000, etc.). With that in mind, it wouldn't match something like `1.2.300-rc1`.

The only thing to note is that this would fail to match a version with fewer than three digits in the last part of the version, like `1.2.00` or `1.2.50`. The Git tags use a minimum of three digits in the last part of the version and seem to have standardized on four nowadays, so it wouldn't be a practical concern yet. However, if we wanted to address that potential issue for the sake of future reliability, we could be more explicit about the final part of the version: `^v?(\d+(?:\.\d+)+\.\d*(?:00|50))$`

Reply:
Thank you again for the explanation, your efforts are very much appreciated.